src/Domain/FunctionsOfTime/Tags.hpp

```diff
@@ -19,11 +21,41 @@ template <size_t VolumeDim>
 class DomainCreator;
 /// \endcond
 
-namespace domain {
-namespace Tags {
+namespace domain::Tags {
+/// \brief The FunctionsOfTime obtained from a DomainCreator or
+/// (if `OverrideCubicFunctionsOfTime` is true) read from a file.
+///
+/// \details When `OverrideCubicFunctionsOfTime` is true, after obtaining the
+/// FunctionsOfTime from the DomainCreator, one or more of those
+/// FunctionsOfTime (which must be cubic piecewise polynomials) is overriden
+/// using data read from an HDF5 file via
+/// domain::Tags::read_spec_third_order_piecewise_polynomial()
+template <size_t Dim, bool OverrideCubicFunctionsOfTime = false>
```

Comment:
This feature looks like it should be decided at runtime, not at compile-time. That could quite easily be done by wrapping the `File` and `NameMap` options in a single option with type `Options::Auto`, replacing the option group you currently have. I feel like it's quick enough and worth the effort to do now instead of in a follow-up PR, though if others think it's OK to defer that's fine with me. @geoffrey4444 if you prefer we can get in a call and implement this fairly quickly some time this week.

Reply:
I'm not convinced that it should be decided at runtime, so let's leave as is for now and discuss at a Monday meeting